moped-editor/src/utils/activityLogHelpers.js

```diff
@@ -0,0 +1,161 @@
+import { useState } from "react";
+import { useQuery, gql } from "@apollo/client";
+import { ProjectActivityLogTableMaps } from "../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const PLACEHOLDER_QUERY = gql`
+  query NotARealQuery {
+    just_a_placeholder {
+      affected_rows
+    }
+  }
+`;
```

Comment:
It does not really matter, but I though this may be easier on the eyes:
https://github.com/cityofaustin/atd-moped/blob/66845ca0d116da02b9a0a93b8a44ec8b45164501/moped-editor/src/components/DataTable/DataTable.js#L63

Reply:
That definitely looks a lot nicer! I'll update with that placeholder. Thanks for pointing out that existing query.